packages/jest-changed-files/src/git.ts

```diff
@@ -72,7 +70,7 @@ const adapter: SCMAdapter = {
     }
   },
 
-  getRoot: async (cwd: string): Promise<?string> => {
+  getRoot: async (cwd: string): Promise<string | null | undefined> => {
```

Comment:
Same here - is it e.g. `Promise<string | void>` or `Promise<string | undefined>`?

Reply:
I've updated it to be `string | null` because `result.stdout` returns a non-nullable string but we return `null` if we catch an error.